### ISSUE TITLE
[A11y] Search Results - List - Linked Image Alternative Text

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1969]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

All images on the "list" search results page are missing an `alt` attribute

### Suggested Fix

In cases of collections, the alt attribute could be null (`alt=""`), because they aren't linked to anything and can be considered decorative as far as I can tell.

All assets' images, however, are links, and need their `alt` attribute to be set. The best value is likely the same as the link next to them, such as `alt="A11y Test 6"`.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

- Verify all images have an `alt` attribute
- Alt attributes must make sense in context, not be a simple description of what's in the image
  - For links, the `alt` attribute should be the page they take you to
  - For images that aren't links, the `alt` attribute should replace the *function* the image serves on the page
